src/database/DataTypes/Location.js

```diff
@@ -73,6 +73,24 @@ export class Location extends Realm.Object {
   get coldConsecutiveBreachConfig() {
     return this.breachConfigs.filtered("type == 'COLD_CONSECUTIVE'")[0];
   }
+
+  get isInHotBreach() {
+    return !!this.breaches.filtered('type == "HOT_CONSECUTIVE" && endTimestamp == null')[0];
+  }
+
+  get isInColdBreach() {
+    return !!this.breaches.filtered('type == "COLD_CONSECUTIVE" && endTimestamp == null')[0];
+  }
```

Comment:
Looking at this now, I think it would have been better to use some sort of 'getCurrentBreach' getter which gets a breach with no `endTimestamp` and then two other getters which organise by type.

Reply:
Bet it will take longer since I'll have to also fix other branches- I'll make a new issue?

Reply:
Sure thang

Reply:
505ca37